CMakeLists.txt

```diff
@@ -71,7 +71,10 @@ install(
     EXPORT ${PROJECT_NAME}-targets)
 
 add_subdirectory(src)
-add_subdirectory(test)
+# do not include tests inside oneCCL
+if(NOT CCL_MAJOR_VERSION)
```

Comment:
I think it would be better to have a generic "UMF_BUILD_TESTS" option that oneCCL (and anyone else) can switch

Reply:
agree - please set this option to true by default and I will disable it in oneCCL

Reply:
Done